mappings/net/minecraft/class_4922.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_4922
+	FIELD field_22855 block Lnet/minecraft/class_2248;
+	METHOD <init> (Lnet/minecraft/class_2248;)V
+		ARG 1 block
+	METHOD method_25758 create (Lnet/minecraft/class_2248;)Lnet/minecraft/class_4922;
+		ARG 0 block
+	CLASS class_4923
+		FIELD field_22857 when Lnet/minecraft/class_4918;
+		METHOD <init> (Lnet/minecraft/class_4918;Ljava/util/List;)V
+			ARG 1 when
+	CLASS class_4924
+		METHOD get getParts ()Ljava/lang/Object;
```

Comment:
🤔

Reply:
`ARG 1 when`

It's a datafixer and Mojang mentions missing parts in conjunction with this method.

Reply:
I am :thinking: on why you renamed `Supplier.get` implementation

Reply:
Because it's not a `Supplier.get` implementation?

Reply:
Welp. Looks like it was a get. Fixed now.